shared/chat/conversation/messages/wrapper/sent.tsx

```diff
@@ -4,7 +4,7 @@ import {Animated} from 'react-native'
 type SentProps = {
   children: React.ReactNode
 }
-export const Sent = function Sent(p: SentProps) {
+export const Sent = React.memo(function Sent(p: SentProps) {
   const {children} = p
```

Comment:
this sent animation is quite slow in debug. i thought it was an issue w/ a library here but its just the debug time between it rendering and us getting an onlayout is quite long. it seems fine in release mode though, same as in v630 and v620 so